source/SUBMIT.rst

```diff
@@ -126,40 +126,38 @@ Example preprint templates using ``repo2data`` for caching data on NeuroLibre se
    * - Nilearn
      - `neurolibre/repo2data-nilearn <https://github.com/neurolibre/repo2data-caching>`_
    * - OSF
-     - `neurolibre/repo2data-osf <https://github.com/ltetrel/neurolibre-osf-test>`_
+     - `neurolibre/repo2data-osf <https://github.com/neurolibre/neurolibre-osf-test>`_
 
 
 
-.. warning:: RoboNeuro may fail downloading relatively large datasets (**exceeding 5GB**) as the book build process times out in 30 minutes.
+.. warning:: 
+  RoboNeuro may fail downloading relatively large datasets (**exceeding 5GB**) as the book build process times out in 60 minutes.
+  If you have data that exceed 5GB, please create an issue in your github repository so a Neurolibre admin can check it.
```

Comment:
So what I discovered for example with https://github.com/neurolibre/ohbm2021-nimare-tutorial
is that we cannot open issue on a fork. It seems to be a recent change from github.. I would be more in favor of your second solution, maybe we could resurect the old `neurolibre/submit` repository for this ?

Reply:
I would not use the word submit just for the semantics of it, I think we can open a new repo names `support` from scratch for such edge case supports.

Reply:
How are we doing the review process if we cannot open on fork? If we open on there repo, we should get notified because we (roboneuro) opened it.

Reply:
> How are we doing the review process if we cannot open on fork? If we open on their repo, we should get notified because we (roboneuro) opened it.

With the new workflow we are not forking author's repo, the reviews are going to happen here https://github.com/neurolibre/neurolibre-reviews. 

We can envision this support repository for people who cannot submit their work using the resources that works fine for everybody else. Because a submission is reuqired to start technical review on neurolibre/neurolibre-reviews.